specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/examples/ApplicationGatewayCreate.json

```diff
@@ -541,6 +541,9 @@
                 "privateIPAllocationMethod": "Dynamic",
                 "publicIPAddress": {
                   "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/publicIPAddresses/appgwpip"
+                },
+                "privateLinkConfiguration": {
```

Comment:
Hi. I'm still a little bit confused. How can user get such a id? We cannot create a privateLinkConfiguration independently.